docs/run-keep-ecdsa.adoc

```diff
@@ -323,6 +323,60 @@ reference].
 |No
 |===
 
+==== Example BeneficiaryAddresses
+
+[%header,cols=2*]
+|===
+|Description
+|Address
+|mainnet xpub extended public key
+|`BeneficiaryAddress = "xpub6Cg41S21V____REPLACE_WITH_VALID_DATA____vZvH8XyNzunEqLdk9dxyXQUoy7ALWQFNn5K1me74aEMtS6pUgNDuCYTTMsJzCAk9sk1"`
```

Comment:
What do you think about removing `BeneficiaryAddress` part from all the entries? The value is soooo long that maybe this will improve how the table looks.

```suggestion
|`xpub6Cg41S21V____REPLACE_WITH_VALID_DATA____vZvH8XyNzunEqLdk9dxyXQUoy7ALWQFNn5K1me74aEMtS6pUgNDuCYTTMsJzCAk9sk1`
```

Reply:
I went back-and-forth on that. I originally didn't include the label and just had the examples. Then, I realized that having the `BeneficiaryAddress = ` removed a lot ambiguity as to what we were referring to and how it should be pasted into the toml. I'm not at all married to the concept though. Maybe at the top of the table we can write a blurb to remove that ambiguity?

Reply:
Vote up for the blurb!